### ISSUE TITLE
the predict action_name accuracy is a little low

### ISSUE THREAD

**Issue by xwr96** [GajuuzZ/Human-Falling-Detect-Tracks#16]
Hi! ths for your great repo. 
but when i use camera to detection fall down, the accuracy is little low. so how can i increase the accuracy? retrain the posemodel or st-gcn model?

**Comment by GajuuzZ** [comment]
Due to the Model is 2D pose, So on same pose of various view should be hard for the model. And the training data set didn't cover all possible camera view. If you testing on camera view that have much different from training set it should be low on accurate.

**Comment by xwr96** [comment]
thanks a lot!
